x-pack/distributions/internal/observabilitySRE/qa/acceptance/docker/elasticsearch/config/elasticsearch-fips.yml

```diff
@@ -0,0 +1,14 @@
+# Elasticsearch settings
+discovery.type: single-node
```

Comment:
I will probably move toward file based ES config for smoke test as well, currently its done mainly in docker compose manifest